tic_tac_toe/board.py

```diff
@@ -4,23 +4,32 @@
 class Board:
     def __init__(self):
         self.board = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+        self.winner = None
```

Comment:
Storing this as separate state seems superfluous to me. Just as you can calculate who's turn it is by examining the state of the board, you could do the same to determine if there is a winner or not (and if so, who it is).

Reply:
Totally agree with this.